sb-utils/src/main/java/org.sunbird/JsonKeys.java

```diff
@@ -51,7 +51,15 @@
     public static final String TYPE = "type";
     public static final String ACCOUNT = "account";
     public static final String key = "key";
-
+    public static final String USER_NAME="userName";
+    public static final String RECIPIENT_TYPE="recipientType";
+    public static final String REASON="reason";
+    public static final String COURSE_COMPLETION="course-completion";
```

Comment:
course-completion need to be camel case same for course-performance

Reply:
@manzarul not required